src/functionalTest/java/uk/gov/hmcts/darts/usermanagement/UserManagementFunctionalTest.java

```diff
@@ -98,22 +106,20 @@ void shouldModifyUser() {
 
     @Test
     void getUserByEmail() {
-        Response createUserResponse = createUser();
-        int userId = new JSONObject(createUserResponse.asString())
-            .getInt("id");
+        int userId = extractId(createUser());
 
         Response getUserResponse = buildRequestWithExternalGlobalAccessAuth()
-            .baseUri(getUri(ADMIN_USERS))
             .contentType(ContentType.JSON)
-            .queryParam(COURTHOUSE_ID, 21)
-            .header(EMAIL_ADDRESS, "james.smith@hmcts.net")
-            .get()
+            .header(EMAIL_ADDRESS_HEADER_NAME, "james.smith@hmcts.net")
+            .queryParam("courthouse_id", 21)
```

Comment:
For my own understanding where does courthouse 21 come from? Its not in the staging db? I guess this is not meant to exist?

Reply:
Actually looking at this courthouse can be removed. Its not part of the endpoint spec.

Reply:
Good spot, have removed.